python/fbprophet/forecaster.py

```diff
@@ -634,6 +647,9 @@ def add_seasonality(
             'prior_scale': ps,
             'mode': mode,
         }
+        if condition_name is not None:
```

Comment:
I know only add condition_name to seasonality dict if condition_name is not None, but could also refactor the code such that condition_name is always present. Any preference?

Reply:
I do have a preference for having it always present, as None if not used. Is there a reason not to? It seems to me like things will be just as clean since we end up doing `is not None` checks anyway.

Reply:
I have changed it now, not important for me. If not present you could avoid always having to define it when creating a seasonality. A fun alternative to dict for this situation is dataclasses https://docs.python.org/3/library/dataclasses.html, but not to common in use yet.